lib/rubocop/cop/rspec/predicate.rb

```diff
@@ -0,0 +1,182 @@
+# encoding: utf-8
+
+module RuboCop
+  module Cop
+    module RSpec
+      # Prefer using predicate matcher instead of truthiness
+      #
+      # @example
```

Comment:
Ditto for the example section. Can you add an example for each style? See https://github.com/nevir/rubocop-rspec/pull/83/files#r74694733